src/yb/tablet/tablet_bootstrap.cc

```diff
@@ -316,13 +316,35 @@ Status ReplayState::UpdateSplitOpId(const ReplicateMsg& msg, const TabletId& tab
 }
 
 void ReplayState::AddEntriesToStrings(const OpIndexToEntryMap& entries,
-                                      std::vector<std::string>* strings) const {
+                                      std::vector<std::string>* strings,
+                                      int limit) const {
```

Comment:
limit -> half_limit

There is no need to support an odd limit here.

Reply:
If you look at the call chain, the limit for AddEntriesToStrings can be odd in current code base.
I don't think one log line would make a difference in terms of the efficacy of the dump, so I can use half_limit.